### PR TITLE
Upper ECAM Fixed wrong condition for thrust modes and engine mode START

### PR DESCRIPTION
**Pull request by MisterChocker**
## Summary of Changes

- Changed condition for engine mode START to fix bug of wrong thrust mode being displayed on inflight engine fail.
- Fixed thrust rating mode condition to get correct thrust rating mode.

## Screenshots (if necessary)

Before:
(Thrust lever in FLX/MCT detent)
image

After:
image

Before:
After liftoff on FLX thrust mode changed immediately to MCT
image

After:
image

## Additional context

Before:
On inflight engine failure displayed thrust mode was CLB regardless of thrust lever position. If set to detent FLX/MCT or TOGA, CLB would be the displayed thrust mode which is wrong.

After:
On inflight engine failure displayed thrust mode shows correct thrust mode for thrust lever position/detent. E.g. Thrust mode will be MCT when thrust lever in FLX/MCT detent or TOGA if in TOGA detent.

Note:
For takeoff phase, corresponding thrust mode will now be displayed as well.

Discord username (if different from GitHub): DerL30N#3751

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by MisterChocker** [comment]
Can I get a ready to test pls?

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js [hunk 1]
```diff
@@ -1878,8 +1878,8 @@ var A320_Neo_UpperECAM;
          * @returns string
          */
         getThrustRatingMode(_grounded, _min = "CLB", _med = "FLX", _max = "TOGA") {
-            if (_grounded) {
-                return ((((Simplane.getCurrentFlightPhase() < FlightPhase.FLIGHT_PHASE_CLIMB) && (Simplane.getFlexTemperature() > 0))) ? _med : _max);
+            if (_grounded || (Simplane.getCurrentFlightPhase() < FlightPhase.FLIGHT_PHASE_CLIMB)) {
+                return ((Simplane.getFlexTemperature() > 0) ? _med : _max);
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js:1882]:
```suggestion
                return Simplane.getFlexTemperature() > 0 ? _med : _max;
```

Comment (MisterChocker) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js]: I do believe at least the condition needs to be within (), but I am not into js

**Comment by ImenesFBW** [comment]
Name: Imenes#8739
Date of testing: 1/11/2020
Version of the sim: 1.10.8.0
PR Tested: #1578
Object of test: Testing upper ECAM thrust modes and engine mode on in-flight failure 


Testing technique: Try-out (multiple tests)
Steps you've made to test the Pull Request:

1. Started cold and dark and flew a full flight  while monitoring modes on the upper ecam
2. Started mid-flight and checked the modes with in-flight engine failure.


Flight notes:

Startup: Working as documented

Taxi/Takeoff : Working as documented

Cruise: Working as documented

Descent/Landing: Working as documented

Shutdown: n/a

Overall Rating: Good

Conclusions of the testing: Fixes are working as described, LGTM! Thank you !

**Comment by Benjozork** [comment]
Simple enough, LGTM

**Pull request merged**